package.json

```diff
@@ -41,6 +42,12 @@
     "minimatch": "^3.0.0",
     "sb-debounce": "^1.0.1"
   },
+  "devDependencies": {
+    "babel-eslint": "^5.0.0",
```

Comment:
If we want to take on a beta dependency, `babel-eslint@6.0.0-beta.6` supports `eslint@^2.4.0`.